packages/shared_preferences/lib/shared_preferences.dart

```diff
@@ -44,6 +44,9 @@ class SharedPreferences {
   /// in sync since the setter method might fail for any reason.
   final Map<String, Object> _preferenceCache;
 
+  /// Reads all values from cache
+  Map<String, Object> getAll() => new Map.from(_preferenceCache);
```

Comment:
This is causing an analysis error (see the travis log, or run `flutter analyze` from your sources). You are missing a type annotation on the right-hand side; see the `getAll()` method for an example.